data/VT/incentive_relationships.json

```diff
@@ -14,10 +14,9 @@
     "VT-86": "VT-84",
     "VT-89": "VT-87",
     "VT-93": "VT-91",
+    "VT-94": {"anyOf": ["VT-75", "VT-77", "VT-79", "VT-81"]},
     "VT-96": {
-      "allOf": [
-        "VT-94",
-        { "anyOf": ["VT-75", "VT-77", "VT-79", "VT-81"] }]
+      "allOf": ["VT-94"]
```

Comment:
Another alternative here could be something like, which would also work downstream in incentive admin... 
```json
    "VT-96": {
      "allOf": ["VT-94"],
      "anyOf": ["VT-75", "VT-77", "VT-79", "VT-81"]
    }
```

Reply:
But, if we go this route we would also need to update the schema to allow for more than oneOf the expected definitions. 
<img width="486" alt="image" src="https://github.com/rewiringamerica/api.rewiringamerica.org/assets/1214752/b41f2837-81f8-47ac-87c3-2b6832bb0aa0">

Reply:
Thinking about this more and I don't think either solution works. I'm going to consider if I can create a recursive solution in Incentive Admin to follow the nesting. Open to ideas though. 
<img width="403" alt="image" src="https://github.com/rewiringamerica/api.rewiringamerica.org/assets/1214752/7ef46d88-21c6-4db2-b6ce-d1d7f7475e2b">